arangod/RocksDBEngine/RocksDBReplicationContext.cpp

```diff
@@ -255,10 +255,10 @@ RocksDBReplicationResult RocksDBReplicationContext::dumpJson(
     TRI_vocbase_t* vocbase, std::string const& cname,
     basics::StringBuffer& buff, uint64_t chunkSize) {
   TRI_ASSERT(_users > 0 && !_exclusive);
-  CollectionIterator* collection{nullptr};
-  TRI_DEFER(if (collection) {
+  CollectionIterator* collectionIter{nullptr};
+  TRI_DEFER(if (collectionIter) {
```

Comment:
according to the comments in Common.h this is not allowed.

Reply:
updated comment